webpack.config.js

```diff
@@ -1,23 +1,30 @@
 const path = require("path");
-const { readFileSync } = require("fs");
 
 module.exports = {
   watch: process.env.WEBPACK_WATCH === "true",
-  entry: "./georaster-layer-for-leaflet.js",
+  entry: "./src/georaster-layer-for-leaflet.ts",
+  mode: "production",
   output: {
     filename: "georaster-layer-for-leaflet.min.js",
-    path: path.resolve(__dirname)
+    path: path.resolve(__dirname, "dist"),
+    library: "libraryStarter",
```

Comment:
shouldn't this be `library: GeoRasterLayer`?
https://webpack.js.org/configuration/output/#outputlibrary

Reply:
Yes definitely. Missed that